compiler/rustc_graphviz/src/lib.rs

```diff
@@ -1,21 +1,19 @@
-//! Generate files suitable for use with [Graphviz](https://www.graphviz.org/)
+//! Generate files suitable for use with [Graphviz](https://graphviz.org/)
```

Comment:
Why did you change? I don't think it's necessary to fix the links below.

Reply:
I am not particular about adding "www" or not, so I changed it according to your policy.

Reply:
I mean, unrelated diffs make it unobvious what's changed and fixed. From that point of view, I didn't request to add subdomains but remove these diffs.

Reply:
Also, it'd be great if you could squash commits once it's done.

Reply:
I appreciate your advice.
And I squashed commits.